### PR TITLE
Update file reference for deploymentconfig

### PR DESCRIPTION
**Pull request by cdrage**
Updates the file reference file with references to deploymentconfig

### CODE REVIEW

File: docs/file-reference.md [hunk 1]
```diff
@@ -161,30 +161,55 @@ name: mariadb
...
 ```
 
+> Example using DeploymentConfig for OpenShift
+
+```yaml
+controller: deploymentconfig
+name: httpd
+replicas: 2
+containers:
+- image: centos/httpd
```
Comment (kadel) [inline docs/file-reference.md]: this container won't work on OpenShift  (root privileges)

Comment (kadel) [reply docs/file-reference.md]: you can use `bitnami/nginx` or `tomaskral/nonroot-nginx` (not sure if there is openshift's or redhat's official image)

Comment (cdrage) [reply docs/file-reference.md]: thanks. updated.

File: docs/file-reference.md [hunk 2]
```diff
@@ -161,30 +161,55 @@ name: mariadb
...
+name: httpd
+replicas: 2
+containers:
+- image: bitnami/nginx
+services:
+- name: httpd
+  type: NodePort
+  ports:
+  - port: 8080
+    targetPort: 80
```
Comment (kadel) [inline docs/file-reference.md]: one more small change. `bitnami/nginx` is listening in `8080`

Comment (cdrage) [reply docs/file-reference.md]: done.

**Pull request merged**